WcaOnRails/app/uploaders/avatar_uploader_base.rb

```diff
@@ -3,6 +3,8 @@
 class AvatarUploaderBase < CarrierWave::Uploader::Base
   include CarrierWave::MiniMagick
 
+  DEFAULT_URL = ActionController::Base.helpers.asset_url("missing_avatar_thumb.png", host: ENVied.ROOT_URL).freeze
```

Comment:
Did you look into why this is "bypassing" the `Rails.application.routes.default_url_options` setting you have in `application.rb`? Maybe there's a different way of calling this that will take advantage of the `default_url_options` you already set elsewhere?